packages/manager/modules/vps/src/migration/components/migration-banner/vps-migration-banner.controller.js

```diff
@@ -0,0 +1,6 @@
+export default class {
```

Comment:
Unnecessary controller.

Reply:
The entire `migration-banner` has to be removed, since it is not used